app/models/recipe.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+class Recipe < ApplicationRecord
+  belongs_to :recipe_category
+  belongs_to :user
+
```

Comment:
[Validations](https://guides.rubyonrails.org/active_record_validations.html#numericality)! Use them to catch through the model any attempt to store undesired values :)